web/src/lib/types/types.ts

```diff
@@ -222,6 +222,15 @@ export interface QuickActionLink extends QuickAction {
   externalRoute: string;
 }
 
+export interface QuickActionLicenseReinstatement extends QuickAction {
+  urlSlug: string;
+  contentMd: string;
+  callToActionLink: string | undefined;
+  callToActionText: string | undefined;
+  form: string | undefined;
+  summaryDescriptionMd: string;
+}
```

Comment:
[sand] I'm not sure if it makes sense for there to be a `QuickActionLicenseReinstatement` type. This is really just the same thing as a `QuickActionTask`. The only difference between these components is that one will conditionally render based on the status of a license.

It looks like this is already merged, but in the future if you feel strongly about creating a new type like this I might have just had `QuickActionLicenseReinstatement` extend `QuickActionTask`.